Common/GPU/Shader.h

```diff
@@ -102,9 +102,9 @@ enum class SamplerFlags {
 ENUM_CLASS_BITOPS(SamplerFlags);
 
 struct SamplerDef {
-	int binding;  // Might only be used by some backends.
-	const char *name;
+	const char* name;
 	SamplerFlags flags;
+	int binding;  // Might only be used by some backends.
```

Comment:
This particular one won't matter for performance, here I prefer the original order, looks better in the table using the struct later.

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/19047/commits/87bd591cee2a17d537b75330467cc699318fd3b9